src/main/java/com/whalespottingjava/services/MemberDetailsService.java

```diff
@@ -28,4 +29,12 @@ public UserDetails loadUserByUsername(String username) {
 
         return new MemberDetails(member);
     }
+
+    public Member loadMemberById(long id) {
+        return memberRepository
+                .findById(id)
+                .orElseThrow(
+                        EntityNotFoundException::new
+                );
+    }
 }
```

Comment:
New Line at end of file

Reply:
There is one at the end of the file, git just doesn't show it. I believe the sign is the no-entry type symbol at the end of the file on the diff